src/utils/beaconPath.ts

```diff
@@ -41,6 +42,11 @@ export class BeaconPath {
     } else {
       support.warns.notSupportedClassNamespacedEndpointsForBeacons();
     }
+    if(support.version){
+      if (isValidWeaviateVersion(support.version)){
```

Comment:
If `isValidWeaviateVersion` changes as per my comment below, this will need to change too 😁